__tests__/app.test.js

```diff
@@ -0,0 +1,38 @@
+const seed = require("../db/seeds/seed");
+const { articleData, commentData, topicData, userData } = require("../db/data/test-data/index");
+const db = require("../db/connection");
+const request = require("supertest");
+const app = require("../db/app");
+
+beforeAll(() => {
+  return seed({ articleData, commentData, topicData, userData });
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+
+describe('GET: /api/topics', () => {
+    it('status: 200, responds with a json object containing a key of `topics` with a value of an array of all the topic objects', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((res) => {
+            expect(res.body).toHaveProperty('topics')
+            expect(Array.isArray(res.body.topics)).toBe(true)
+            res.body.topics.forEach((topic) => {
```

Comment:
Just before this forEach we should check the length of the response, which we can figure out by looking at the test data, because if the response is length 0 then the forEach would never trigger so we could get false positives